README.md

```diff
@@ -62,47 +113,29 @@ HexDigits ::
 
 ### Standard library
 
-We want to make sure that this syntax is consistent with the usage of the standard library. This will involve making libraries like the following compatible with the proposed parsing rules:
+We want to make sure that this syntax is consistent with the usage of the standard library. `Number()` will accept string values that safely convert via `ToNumber()` and ignore occurrences of _NumericLiteralSeparator_. `parseInt()` and `parseFloat()` will remain unchanged. 
```

Comment:
Should we make a note that this leads to a non-backwards-compatible change to Number()?